pkg/operation/common/types.go

```diff
@@ -162,6 +162,15 @@ const (
 	// GrafanaUsersPrefix is a constant for a prefix used for the users Grafana instance.
 	GrafanaUsersPrefix = "gu"
 
+	// PrometheusPrefix is a constant for a prefix used for the Prometheus instance.
+	PrometheusPrefix = "p"
+
+	// AlertManagerPrefix is a constant for a prefix used for the AlertManager instance.
+	AlertManagerPrefix = "au"
```

Comment:
```suggestion
	AlertManagerPrefix = "a"
```
Why not only `a` as before?

Reply:
The ingress has had "au"
https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/monitoring.go#L209
@wyb1